job_controller/job.go

```diff
@@ -191,6 +191,16 @@ func (jc *JobController) ReconcileJobs(
 		return jc.Controller.UpdateJobStatusInApiServer(job, &jobStatus)
 	}
 
+	if jc.Config.EnableGangScheduling {
+		minAvailableReplicas := getTotalReplicas(replicas)
+		priorityClassName := getPriorityClassName(runPolicy)
+		//_, err := pc.SyncPodGroup(job, minAvailableReplicas)
```

Comment:
why keep this ?

Reply:
I reference the code about EnableGangScheduling of other operators and add the priority‘s’attribute and function.